README.md

```diff
@@ -1,103 +1,47 @@
+Components should be added to this gem if they are required in more than one application, otherwise they should be added to that application. All components should meet the same standards of browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
 
-## Relationship with GOV.UK Design System
+- 👉 [Install and use this gem](docs/install-and-use.md)
+- 👉 [Use a component in your application](docs/use-components.md)
+- 👉 [Develop a component](docs/develop-component.md)
 
-GOV.UK Publishing Components also makes [GOV.UK Design System](https://design-system.service.gov.uk/) styles and components available to GOV.UK's frontend applications. This gem consumes [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) via [Yarn](https://classic.yarnpkg.com/).
+## Technical documentation
 
-## Browser and assistive technology support
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
-GOV.UK Publishing Components shares the same standards in terms of browser and assistive technology support with [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
 
-## Documentation
-
-[See the rubydoc.info documentation](http://www.rubydoc.info/gems/govuk_publishing_components)
-
-## Working locally
-
-### Install dependencies
-
-As well as the Ruby, you will also need the [Yarn package manager](https://classic.yarnpkg.com/en/docs/install/) to allow installing [npm](https://npmjs.com) dependencies.
-
-Install the application's dependencies with:
-
-```sh
-bundle install
-yarn install
-```
-
-### Running a local app with a local gem
-
-The GDS Developer Docs has a [guide on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) that covers setting up a local app to use the local version of the components gem in Docker and using the `./startup.sh` script.
-
-### Running tests
-
-The default rake task runs all the linting and test tasks:
+### Running the test suite
 
 ```sh
+# run all linting / tests
 bin/rake
-```
-Javascript is tested using Jasmine and the [Jasmine gem](https://github.com/pivotal/jasmine-gem). Tests can be run either in the browser or on the command line via the dummy app’s tasks:
 
-```sh
-# browser
+# run JS tests in browser
 bin/rake app:jasmine
 
-# command line
+# run JS tests on command line
```

Comment:
We seem to phrase these rather inconsistently in various applications (for example in `collections`, used as a reference for this PR we have 'To run JavaScript tests (only):'

Reply:
I've had a look to see which READMEs we have this in, which are just: [content-publisher](https://github.com/alphagov/content-publisher#running-the-test-suite), [collections](https://github.com/alphagov/collections#running-the-test-suite) and [whitehall](https://github.com/alphagov/whitehall#running-the-test-suite). I know there are other apps with JS tests, but it's beyond the scope of the work we're doing to fix that. As long as the README is at least correct, someone else can add guidance if they feel the need in future.

Back to your point here, the more popular form is text + code (vs. comment + code), so I've added another commit to switch to that form (2784152f2c7c5e7b09a5b9c51092e10c6a93d60f), as well as a PR to make content-publisher consistent (https://github.com/alphagov/content-publisher/pull/2355), if you'd like to review that as well.

There's actually a bigger problem here, which is that the browser usage of Jasmine doesn't work in GOV.UK Docker. Luckily, @injms has got a PR to try and address that (https://github.com/alphagov/govuk-docker/pull/498) for this repo, so I don't think it needs to block this change. (We'll want to do a similar thing for other apps, too.)